src/TerraformPlanCommand.groovy

```diff
@@ -1,19 +1,21 @@
-class TerraformPlanCommand implements TerraformCommand, Resettable {
+class TerraformPlanCommand implements TerraformCommand, Pluggable<TerraformPlanCommandPlugin>, Resettable {
     private static final DEFAULT_PLUGINS = []
     private boolean input = false
-    private String terraformBinary = "terraform"
     private String command = "plan"
-    String environment
     private prefixes = []
     private suffixes = []
     private arguments = []
-    private static plugins = DEFAULT_PLUGINS.clone()
-    private appliedPlugins = []
     private String directory
     private String errorFile
     private Closure variablePattern
     private Closure mapPattern
 
+    // The static initializer is needed to initialize the static variable inherited
+    // from the Pluggable trait when the class is loaded.
+    static {
+        this.plugins = DEFAULT_PLUGINS.clone()
+    }
+
```

Comment:
Sorry about the above.  Default plugins should be handled in a different way.  If issue #327 were completed first, you wouldn't have to deal with this problem.

I might consider closing out #327 first - I don't like the idea of reaching into the private variables of a trait.

Reply:
Also `this.plugins = X` is calling the public `Pluggable.setPlugins()` method, not modifying the private variable directly.

Reply:
Similar to the comment above, it's less about using a get/set method, and more about reaching into a trait to pull out data and operate on that data.  

The trait `Pluggable` encapsulates behavior around plugins.  If anything, your change suggests that resetting the value of the plugins feels like a good candidate behavior to consider pushing into the trait.

Reply:
It's less about using get/set methods, and more about extracting data out of an internal class to operate on it (similar to the comment above about `terraformBinary`).

Again, this is less about what you did, and more that you've exposed an underlying problem of the library.  DefaultPlugins are treated as a special case, and they shouldn't be. If default plugins were treated like any other plugin (and added through `addPlugin`), we wouldn't be having this conversation, and you wouldn't need the `static{}` line in there.